cluster/selfplay/cc-Dockerfile

```diff
@@ -22,6 +22,7 @@ RUN apt-get install -y --no-install-recommends \
         unzip \
         zip \
         zlib1g-dev \
+        gdb \
```

Comment:
rm? Isn't gdb for debugging?

Reply:
yup.  I ended up debugging some segfaults this way.